classes/order/OrderDetail.php

```diff
@@ -706,7 +706,7 @@ protected function setDetailProductPrice(Order $order, Cart $cart, $product)
             (int) $product['id_product'],
             true,
             ($product['id_product_attribute'] ? (int) ($product['id_product_attribute']) : null),
-            2,
+            6,
```

Comment:
I might be wrong, but should'nt we use `Context::getContext()->getComputingPrecision()` ?

Reply:
ping @PrestaEdit

Reply:
As the rest of the code use it as it, I will say : no ?